### PR TITLE
Removing unversioned docs links from the enterprise folder 

### PR DESCRIPTION
**Pull request by Joelkw**
Pulling them out of /enterprise first. [Context here](https://sourcegraph.slack.com/archives/C0C324C91/p1641855680027300) .

Will have a followup PR for links outside the enterprise folder.

### CODE REVIEW

File: client/web/src/enterprise/batches/list/BatchChangeListPage.tsx [hunk 1]
```diff
@@ -224,7 +224,7 @@ export const NamespaceBatchChangeListPage: React.FunctionComponent<NamespaceBatc
     )
 }
 
-interface BatchChangeListEmptyElementProps extends Pick<BatchChangeListPageProps, 'location' | 'canCreate'> {}
+interface BatchChangeListEmptyElementProps extends Pick<BatchChangeListPageProps, 'location' | 'canCreate'> { }
```
Comment (Joelkw) [inline client/web/src/enterprise/batches/list/BatchChangeListPage.tsx:227]:
```suggestion
interface BatchChangeListEmptyElementProps extends Pick<BatchChangeListPageProps, 'location' | 'canCreate'> {}
```

Comment (Joelkw) [inline client/web/src/enterprise/batches/list/BatchChangeListPage.tsx]:
```suggestion
interface BatchChangeListEmptyElementProps extends Pick<BatchChangeListPageProps, 'location' | 'canCreate'> {}
```

File: client/web/src/enterprise/batches/settings/CodeHostConnections.tsx [hunk 2]
```diff
@@ -41,14 +41,14 @@ export const CodeHostConnections: React.FunctionComponent<CodeHostConnectionsPro
...
             userID
                 ? queryUserBatchChangesCodeHosts({
-                      user: userID,
-                      first: args.first ?? null,
-                      after: args.after ?? null,
-                  })
+                    user: userID,
+                    first: args.first ?? null,
+                    after: args.after ?? null,
+                })
```
Comment (Joelkw) [inline client/web/src/enterprise/batches/settings/CodeHostConnections.tsx:47]:
```suggestion
                      user: userID,
                      first: args.first ?? null,
                      after: args.after ?? null,
                  })
```

File: client/web/src/enterprise/batches/settings/CodeHostConnections.tsx [hunk 3]
```diff
@@ -46,9 +46,9 @@ export const CodeHostConnections: React.FunctionComponent<CodeHostConnectionsPro
                       after: args.after ?? null,
                   })
                 : queryGlobalBatchChangesCodeHosts({
-                      first: args.first ?? null,
-                      after: args.after ?? null,
-                  }),
+                    first: args.first ?? null,
+                    after: args.after ?? null,
+                }),
```
Comment (Joelkw) [inline client/web/src/enterprise/batches/settings/CodeHostConnections.tsx:51]:
```suggestion
                      first: args.first ?? null,
                      after: args.after ?? null,
                  }),
```

File: client/web/src/enterprise/insights/pages/dashboards/creation/InsightsDashboardCreationPage.tsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ import {
 } from './components/insights-dashboard-creation-content/InsightsDashboardCreationContent'
 import styles from './InsightsDashboardCreationPage.module.scss'
 
-interface InsightsDashboardCreationPageProps extends TelemetryProps {}
+interface InsightsDashboardCreationPageProps extends TelemetryProps { }
```
Comment (Joelkw) [inline client/web/src/enterprise/insights/pages/dashboards/creation/InsightsDashboardCreationPage.tsx:22]:
```suggestion
interface InsightsDashboardCreationPageProps extends TelemetryProps {}
```

File: client/web/src/enterprise/insights/pages/insights/creation/intro/IntroCreationPage.tsx [hunk 5]
```diff
@@ -20,7 +20,7 @@ import {
 } from './cards/InsightCards'
 import styles from './IntroCreationPage.module.scss'
 
-interface IntroCreationPageProps extends TelemetryProps {}
+interface IntroCreationPageProps extends TelemetryProps { }
```
Comment (Joelkw) [inline client/web/src/enterprise/insights/pages/insights/creation/intro/IntroCreationPage.tsx:23]:
```suggestion
interface IntroCreationPageProps extends TelemetryProps {}
```

File: client/web/src/enterprise/searchContexts/CreateSearchContextPage.tsx [hunk 6]
```diff
@@ -25,10 +25,10 @@ import { SearchContextForm } from './SearchContextForm'
...
 export interface CreateSearchContextPageProps
     extends RouteComponentProps,
-        ThemeProps,
-        TelemetryProps,
-        Pick<SearchContextProps, 'createSearchContext' | 'deleteSearchContext'>,
-        PlatformContextProps<'requestGraphQL'> {
+    ThemeProps,
+    TelemetryProps,
+    Pick<SearchContextProps, 'createSearchContext' | 'deleteSearchContext'>,
+    PlatformContextProps<'requestGraphQL'> {
```
Comment (Joelkw) [inline client/web/src/enterprise/searchContexts/CreateSearchContextPage.tsx:31]:
```suggestion
        ThemeProps,
        TelemetryProps,
        Pick<SearchContextProps, 'createSearchContext' | 'deleteSearchContext'>,
        PlatformContextProps<'requestGraphQL'> {
```

File: client/web/src/enterprise/searchContexts/SearchContextsListPage.tsx [hunk 7]
```diff
@@ -15,10 +15,10 @@ import { SearchContextsListTab } from './SearchContextsListTab'
...
 export interface SearchContextsListPageProps
     extends Pick<
-            SearchContextProps,
-            'fetchSearchContexts' | 'fetchAutoDefinedSearchContexts' | 'getUserSearchContextNamespaces'
-        >,
-        PlatformContextProps<'requestGraphQL'> {
+    SearchContextProps,
+    'fetchSearchContexts' | 'fetchAutoDefinedSearchContexts' | 'getUserSearchContextNamespaces'
+    >,
+    PlatformContextProps<'requestGraphQL'> {
```
Comment (Joelkw) [inline client/web/src/enterprise/searchContexts/SearchContextsListPage.tsx:21]:
```suggestion
             SearchContextProps,
             'fetchSearchContexts' | 'fetchAutoDefinedSearchContexts' | 'getUserSearchContextNamespaces'
        >,
        PlatformContextProps<'requestGraphQL'> {
```

File: client/web/src/enterprise/site-admin/SiteAdminAuthenticationProvidersPage.tsx [hunk 8]
```diff
@@ -63,9 +63,9 @@ const authProviderFragment = gql`
     }
 `
 
-interface Props extends RouteComponentProps<{}> {}
+interface Props extends RouteComponentProps<{}> { }
```
Comment (Joelkw) [inline client/web/src/enterprise/site-admin/SiteAdminAuthenticationProvidersPage.tsx:66]:
```suggestion
interface Props extends RouteComponentProps<{}> {}
```

File: client/web/src/enterprise/site-admin/SiteAdminAuthenticationProvidersPage.tsx [hunk 9]
```diff
@@ -63,9 +63,9 @@ const authProviderFragment = gql`
...
 
-class FilteredAuthProviderConnection extends FilteredConnection<GQL.IAuthProvider> {}
+class FilteredAuthProviderConnection extends FilteredConnection<GQL.IAuthProvider> { }
```
Comment (Joelkw) [inline client/web/src/enterprise/site-admin/SiteAdminAuthenticationProvidersPage.tsx:68]:
```suggestion
class FilteredAuthProviderConnection extends FilteredConnection<GQL.IAuthProvider> {}
```

File: client/web/src/enterprise/batches/list/BatchChangeListPage.tsx [hunk 10]
```diff
@@ -36,8 +36,8 @@ import { NewBatchChangeButton } from './NewBatchChangeButton'
 
 export interface BatchChangeListPageProps
     extends TelemetryProps,
-        Pick<RouteComponentProps, 'location'>,
-        SettingsCascadeProps<Settings> {
+    Pick<RouteComponentProps, 'location'>,
+    SettingsCascadeProps<Settings> {
```
Comment (Joelkw) [inline client/web/src/enterprise/batches/list/BatchChangeListPage.tsx:40]:
```suggestion
        Pick<RouteComponentProps, 'location'>,
        SettingsCascadeProps<Settings> {
```

**Comment by Joelkw** [comment]
@vovakulikov do I need to worry about the percy review? It says the screenshots don't exist?

**Comment by vovakulikov** [comment]
@Joelkw I think it says this cause the main pipeline failed https://buildkite.com/sourcegraph/sourcegraph/builds/127457 It's serious though, just a prettier run. Run yarn prettier and commit those changes that prettier made.

**Review by unclejustin — APPROVED** [review]

**Comment by Joelkw** [comment]
@unclejustin or @vovakulikov can you either add me to our percy or approve the changes? looks like some screenshots still don't exist but the build succeeded?

**Comment by Joelkw** [comment]
bump here @unclejustin @vovakulikov can you add me to percy and/or resolve these for me?

**Review by vovakulikov — APPROVED** [review]
@Joelkw Sorry it slipped from me. I resolved conflicts with main and run prettier to format all changes in this PR. It should be fine now with Percy and other parts of the CI pipeline. Feel free to merge this one

**Review by eseliger — APPROVED** [review]
I wonder how hard it would be to have an eslint rule that enforces this 🤔

**Pull request merged**